mousestyles/classification/classification.py

```diff
@@ -0,0 +1,127 @@
+from sklearn.grid_search import GridSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+
+import numpy as np
+import pandas as pd
+
+
+def prep_data(strain, features, rseed=222):
+    """
+    Returns a list of 4: [train_y, train_x, test_y, test_x]
+        train_y: list of strain labels in train data sets,
+        train_x: list of features in train data sets,
+        test_y: list of strain labels in test data sets,
+        test_x: list of features in train data sets
+    Parameters
+    ----------
+    strain: ndarray, shape of (1921,)
+            classification labels
+    features: ndarray, shape of (1921, 99)
+              classification features
+    rseed: int, optional
+           random seed for shuffling the data set to separate train and test
+    Returns
+    ----------
+    The list as specified
+    """
+    # total: 21131 rows
+    index = np.arange(features.shape[0])
+    np.random.seed(rseed)
+    np.random.shuffle(index)
+    # split the dataset to 75% training and 25% testing
+    sep = int(features.shape[0] * 0.75)
+    # total 15848 rows
+    train_x = features.iloc[index[:sep]]
+    train_y = strain.iloc[index[:sep]]
+    # total 5283 rows
+    test_x = features.iloc[index[sep:]]
+    test_y = strain.iloc[index[sep:]]
+    return [train_y, train_x, test_y, test_x]
+
+
+def RandomForest(strain, features, rseed=222):
+    """
+    Returns a ndarray of RandomForest results, containing prediction strain
+    labels and true strain labels for test data set. Use cross validation
+    to tune Parameters.
+    Parameters
+    ----------
+    strain: ndarray, shape of (1921,)
+            classification labels
+    features: ndarray, shape of (1921, 99)
```

Comment:
Can we let the user decide the number of features as input? For example, they might just input 3*11 features for the RF algorithm